src/Components/IssuesControl.test.jsx

```diff
@@ -33,23 +33,36 @@ describe('IssueControl', () => {
     expect(await getByText('open_workspace')).toBeInTheDocument()
     expect(await getByText('closed_system')).toBeInTheDocument()
   })
-})
-test('Setting comments in zustand ', async () => {
-  const {result} = renderHook(() => useStore((state) => state))
-  const testIssueId = 10
-  const {getByText} = render(<ShareMock><Issues/></ShareMock>)
-  await act(() => {
-    result.current.setSelectedIssueId(testIssueId)
-  })
-  await act(() => {
-    result.current.setIssues(MOCK_ISSUES)
+
+  it('Setting comments in zustand ', async () => {
+    const {result} = renderHook(() => useStore((state) => state))
+    const testIssueId = 10
+    const {getByText} = render(<ShareMock><Issues/></ShareMock>)
+    await act(() => {
+      result.current.setSelectedIssueId(testIssueId)
+    })
+    await act(() => {
+      result.current.setIssues(MOCK_ISSUES)
+    })
+    await act(() => {
+      result.current.setComments(MOCK_COMMENTS)
+    })
+    expect(await getByText('open_workspace')).toBeVisible()
+    // expect(await getByText('The Architecture, Engineering and Construction')).toBeVisible()
+    // expect(await getByText('Email is the medium that still facilitates major portion of communication')).toBeVisible()
   })
-  await act(() => {
-    result.current.setComments(MOCK_COMMENTS)
+
+  it('test Loader', async () => {
+    const {result} = renderHook(() => useStore((state) => state))
```

Comment:
@pablo-mayrgundter @oo-bldrs 
need help with this test. 
in order to test if the loader appears in the DOM, a request to get the issues need to take some time, or perhaps the request needs to fail, I am not sure how to simulate a failed request.
Please help.